src/data/externalTutorials.json

```
@@ -1,13 +1,98 @@
 [
+  {
+    "url": "https://youtu.be/wUjYK5gwNZs",
+    "title": "Learn Solidity, Blockchain Development, & Smart Contracts | Powered By AI - Full Course (12 - 15)",
+    "description": "This course will give you a full introduction to all of the core concepts related to blockchain, smart contracts, Solidity, ERC20s, Foundry, DeFi, Chainlink, Ethereum, upgradable smart contracts, DAOs, IPFS, and more. Follow along with the videos and you'll be a blockchain wizard in no time!",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": [
+      "solidity",
+      "smart contracts",
+      "video",
+      "oracles",
+      "NFT",
+      "ERC-20",
+      "foundry",
+      "mocking",
+      "testing",
+      "defi",
+      "openzeppelin",
+      "slither",
+      "storage",
+      "alchemy",
+      "block explorer",
+      "fuzzing"
+    ],
+    "skillLevel": "beginner",
+    "timeToRead": "495",
+    "lang": "en",
+    "publishDate": "06/07/2023"
+  },
+  {
+    "url": "https://youtu.be/sas02qSFZ74",
+    "title": "Learn Solidity, Blockchain Development, & Smart Contracts | Powered By AI - Full Course (7 - 11)",
+    "description": "This course will give you a full introduction to all of the core concepts related to blockchain, smart contracts, Solidity, ERC20s, Foundry, DeFi, Chainlink, Ethereum, upgradable smart contracts, DAOs, IPFS, and more. Follow along with the videos and you'll be a blockchain wizard in no time!",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": [
+      "solidity",
+      "smart contracts",
+      "video",
+      "oracles",
+      "NFT",
+      "ERC-20",
+      "foundry",
+      "mocking",
+      "testing",
+      "defi",
+      "openzeppelin",
+      "slither",
+      "storage",
+      "alchemy",
+      "block explorer",
+      "fuzzing"
+    ],
+    "skillLevel": "beginner",
+    "timeToRead": "640",
+    "lang": "en",
+    "publishDate": "06/07/2023"
+  },
+  {
+    "url": "https://youtu.be/umepbfKp5rI",
+    "title": "Learn Solidity, Blockchain Development, & Smart Contracts | Powered By AI - Full Course (0 - 6)",
```

Changed re: https://github.com/ethereum/ethereum-org-website/pull/10395/files#r1259668384

Also, I don't know what 'Powered by AI' means in this context, but it doesn't feel particularly relevant here.

```suggestion
    "title": "Learn Solidity, Blockchain Development, & Smart Contracts",
```